sr_ronex_controllers/src/spi_sensor_read_controller.cpp

```diff
@@ -161,7 +161,16 @@ void SPISensorReadController::update(const ros::Time& time, const ros::Duration&
     }
     command_queue_[*channel_iter].pop();
   }
-  sensor_data_publisher_.publish(sensor_msg_);
+
+  if (publisher_counter_ % 10 == 0)
```

Comment:
For the publisher of a sensor it might be worth using [this](https://github.com/shadow-robot/sr-ros-interface-ethercat/blob/indigo-devel/sr_tactile_sensor_controller/src/sr_pst_tactile_sensor_publisher.cpp#L31) to control the publish rate. (Maybe in another PR)